code/modules/food&drinks/kitchen machinery/deep_fryer.dm

```diff
@@ -34,12 +38,22 @@
 		return
 	on = TRUE
 	F.loc = src
-	frying = F
```

Comment:
BYOND is complaining about this being removed, AKA Frying doesn't exist in current context for line 57